code/modules/admin/topic.dm

```diff
@@ -3036,25 +3036,50 @@
 				feedback_inc("admin_secrets_fun_used",1)
 				feedback_add_details("admin_secrets_fun_used","CA")
 				var/answer = alert("Are you sure you want to create a custom artifact?",,"Yes","No")
+
 				if(answer == "Yes")
 					//Either have them as all random, or have custom artifacts
 					var/list/effects = typesof(/datum/artifact_effect)
+					var/list/triggers = typesof(/datum/artifact_trigger)
 					effects.Remove(/datum/artifact_effect)
+					triggers.Remove(/datum/artifact_trigger)
+
 					var/answer1 = alert("Just a primary, or primary and secondary effects?",,"Primary only","Primary and Secondary")
-					var/primary_effect = input(usr, "Which primary effect would you like?", "Primary effect") as null|anything in effects
-					var/secondary_effect
+					var/answer2 = alert("Randomly generated triggers (safer), or manually picked (might break certain effects)?",,"Random","Manual")
```

Comment:
Copy pasted from the original code, I assume there's an argument that's by default there.

I can shove a null or something there if you want.